src/Semantic/Task.hs

```diff
@@ -106,41 +80,13 @@ type TaskEff
   ( DistributeC
   ( LiftC IO)))))))))))
 
--- | A function to render terms or diffs.
-type Renderer i o = i -> o
-
--- | A task which parses a 'Blob' with the given 'Parser'.
-parse :: (Member Task sig, Carrier sig m)
-      => Parser term
-      -> Blob
-      -> m term
-parse parser blob = send (Parse parser blob pure)
-
--- | A task which decorates a 'Term' with values computed using the supplied 'RAlgebra' function.
-decorate :: (Functor f, Member Task sig, Carrier sig m)
-         => RAlgebra (TermF f Loc) (Term f Loc) field
-         -> Term f Loc
-         -> m (Term f field)
-decorate algebra term = send (Decorate algebra term pure)
-
--- | A task which diffs a pair of terms using the supplied 'Differ' function.
-diff :: (Diffable syntax, Eq1 syntax, Hashable1 syntax, Traversable syntax, Member Task sig, Carrier sig m)
-     => These (Term syntax ann) (Term syntax ann)
-     -> m (Diff syntax ann ann)
-diff terms = send (Semantic.Task.Diff terms pure)
-
--- | A task which renders some input using the supplied 'Renderer' function.
-render :: (Member Task sig, Carrier sig m)
```

Comment:
Replaced with pure function application in the approximately two places this was even used.